app/views/admin/users/index.html.haml

```diff
@@ -1,10 +1,14 @@
 .row
   .col-md-12
     .page-header
-      %h2
+      %h1
         Users
         - if @users
           = "(#{@users.length})"
+
+        - if can? :create, User and ENV['OSEM_ICHAIN_ENABLED'] != 'true'
+          .pull-right
+            =link_to 'Add User', new_admin_user_path, :class => 'button btn btn-default btn-info'
```

Comment:
@ChrisBr we had a discussion on that with @differentreality some time ago, please see this PR https://github.com/openSUSE/osem/pull/1257
long story short - it is nice to have a visually separated primary and secondary actions from the UX perspective, I used the same approach in the https://github.com/openSUSE/osem/pull/1278
If you guys strongly against that, I would not argue and change the btn to the default green, but I think it is a good step toward more user-friendlt UI